src/utilities/controllers/discon_rotor_test_controller.cpp

```diff
@@ -11,8 +11,7 @@ namespace openturbine::util {
 extern "C" {
 
 void PITCH_CONTROLLER(
-    float avrSWAP[], int* aviFAIL, const char* const accINFILE, char* const avcOUTNAME,
-    char* const avcMSG
+    float avrSWAP[], int* aviFAIL, const char* const, char* const, char* const avcMSG
```

Comment:
This change also seems a little random requirement from the static analyzer.

Reply:
Not providing names to unused variables in function signatures is pretty common.  I've had to make this change in other code bases due to warnings from certain compiler+platform combinations even when not using ClangTidy.  Now that you mention it, I'm surprised our compiler warnings don't flag it.

Reply:
I thought we got warnings for unused variables, didn't realize deleting the names would be the solution. Anyways, just pointing out this feels like a little random, which looks like you agree with.